src/EventTickets/Routes/GetEventTicketTypeTickets.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Give\EventTickets\Routes;
+
+use Give\API\RestRoute;
+use Give\EventTickets\Models\Event;
+use Give\EventTickets\Models\EventTicket;
+use Give\EventTickets\Models\EventTicketType;
+use Give\Framework\Models\Model;
+use WP_REST_Request;
+use WP_REST_Response;
+
+/**
+ * @unreleased
+ */
+class GetEventTicketTypeTickets implements RestRoute
+{
+    /** @var string */
+    protected $endpoint = 'events-tickets/ticket-type/(?P<ticket_type_id>\d+)/tickets';
+
+    /**
+     * @inheritDoc
+     */
+    public function registerRoute()
+    {
+        register_rest_route(
+            'give-api/v2',
+            $this->endpoint,
+            [
+                [
+                    'methods' => 'GET',
+                    'callback' => [$this, 'handleRequest'],
+                    'permission_callback' => '__return_true',
+                ],
+                'args' => [
+                    'ticket_type_id' => [
+                        'type' => 'integer',
+                        'sanitize_callback' => 'absint',
+                        'validate_callback' => function ($ticketTypeId) {
+                            return EventTicketType::find($ticketTypeId);
+                        },
+                        'required' => true,
+                    ],
+                    'page' => [
+                        'validate_callback' => function ($param) {
+                            return filter_var($param, FILTER_VALIDATE_INT);
+                        },
+                        'default' => 1,
+                    ],
+                    'per_page' => [
+                        'validate_callback' => function ($param) {
+                            return filter_var($param, FILTER_VALIDATE_INT);
+                        },
+                        'default' => 10,
+                    ],
+                ],
+            ]
+        );
+    }
+
+    /**
+     * @unreleased
+     *
+     * @return WP_REST_Response
+     *
+     */
+    public function handleRequest(WP_REST_Request $request)
+    {
+        $tickets = EventTicket::findByTicketType($request->get_param('ticket_type_id'))
+            ->paginate(
+                $request->get_param('page'),
+                $request->get_param('per_page')
+            )
+            ->getAll();
+
+        return new WP_REST_Response(
```

Comment:
The REST API for ListTable returns an instance of a ListTable object. This object includes headers, columns, and formatted values that can be printed by the React app. Unless these routes were not intended to be used on that.

Reply:
Let's include these REST Endpoints as is for regular data lookup and I'll add List Table specific endpoints in a new PR.

Reply:
See [GIVE-289](https://stellarwp.atlassian.net/browse/GIVE-289?atlOrigin=eyJpIjoiN2QyYmJhYzJjYTBmNGFjMWE3YTVkNGIyNjVjNDIxODciLCJwIjoiaiJ9).